src/main/java/seedu/address/ui/CommandBox.java

```diff
@@ -53,19 +57,17 @@ private void handleCommandEntered() {
      * Sets the command box style to use the default style.
      */
     private void setStyleToDefault() {
-        commandTextField.getStyleClass().remove(ERROR_STYLE_CLASS);
+        commandTextArea.getStyleClass().remove(ERROR_STYLE_CLASS);
     }
 
     /**
      * Sets the command box style to indicate a failed command.
      */
     private void setStyleToIndicateCommandFailure() {
-        ObservableList<String> styleClass = commandTextField.getStyleClass();
-
+        ObservableList<String> styleClass = commandTextArea.getStyleClass();
         if (styleClass.contains(ERROR_STYLE_CLASS)) {
```

Comment:
ERROR_STYLE_CLASS might need to be adjusted for a TextArea, not sure but worth for double checking!

Reply:
The text itself is affected by the ERROR_STYLE_CLASS, so LGTM